corehq/apps/export/utils.py

```diff
@@ -282,6 +294,61 @@ def _get_for_single_node_repeat(tables, column_path, transform):
             return new_column
 
 
+def _get_column_for_stock_form_export(new_table, column_path, index):
+    # Takes a path like column.transfer:question_id.@date
+    # and maps it to column.transfer.@date
+    def _remove_question_id_from_path(path):
+        parts = path.split('.')
+        parts_without_question_ids = map(lambda part: part.split(':')[0], parts)
+        return '.'.join(parts_without_question_ids)
+
+    stock_columns = filter(lambda c: c.item.doc_type == 'StockItem', new_table.columns)
+
+    # Map column to its readable path (dot path)
+    stock_column_to_readable_path = {c: c.item.readable_path for c in stock_columns}
+
+    matched_columns = []
+    for column, readable_path in stock_column_to_readable_path.iteritems():
+        if _remove_question_id_from_path(readable_path) == index:
+            matched_columns.append(column)
+
+    if len(matched_columns) == 1:
+        return matched_columns[0]
+    elif len(matched_columns) == 0:
+        return None
+    else:
+        raise SkipConversion('Multiple matched stock nodes')
```

Comment:
this is the scenario where there exists multiple stock questions at the same level